### PR TITLE
Lecturer can add courses - it's magic!

### PR DESCRIPTION
**Pull request by plusk**

### CODE REVIEW

File: iris/views.py [hunk 1]
```diff
@@ -150,6 +150,22 @@ def handle_lecturer_send(message):
     db.session.commit()
 
 
+@socketio.on('lecturer_course_new_send')
+def handle_lecturer_course_new(message):
+    if not current_user.is_authenticated:
```
Comment (eirism) [inline iris/views.py]: Needs docstring

**Comment by plusk** [comment]
I have no idea what just happened, but I think it's fixed now.

File: iris/static/js/lecturer.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+})
+
+socket.on('lecturer_course_new_recv', function (msg) {
+  let code = msg['code']
+  let name = msg['name']
+
+  $('#table_courses').children().append(
+    '<tr>' +
+      '<td class="table_row mdl-data-table__cell--non-numeric">' +
+        '<a class="course_links" href="/lecturer/{code}/session">' + code + ' - ' + name + '</a>' +
```
Comment (eirism) [inline iris/static/js/lecturer.js]: The actual course code should be used in the link, currently it doesn't replace {code}

**Review by eirism — DISMISSED** [review]

File: iris/static/js/lecturer.js (hunk 2 above)
Comment (plusk) [reply iris/static/js/lecturer.js]: That one is on me, been working a lot with JSX lately, easy to get syntax mixed up. Will fix imminently.

**Review by Mattis52 — APPROVED** [review]

**Pull request merged**